.env.development

```diff
@@ -3,5 +3,5 @@ NEXT_PUBLIC_RPC_URL=https://kovan.infura.io/v3/092108ec6aea46ab97b2175b45130455
 NEXT_PUBLIC_ENV=demo
 NEXT_PUBLIC_TRANSACTION_TIMEOUT=3600
 NEXT_PUBLIC_TINLAKE_DATA_BACKEND_URL=https://api.thegraph.com/subgraphs/name/centrifuge/tinlake-kovan-staging
-NEXT_PUBLIC_POOLS=[ {"addresses":{"ROOT_CONTRACT":"0x41b7b379dee711b1a9bfbabd4b1309a584f5fe5a", "ACTIONS":"0x60cc4a2868559e513112f0742e6f546d339ea17b", "PROXY_REGISTRY":"0x99a31c86d95e907dcf0001857b8aa83456e746c1", "COLLATERAL_NFT": "0xac0c1ef395290288028a0a9fdfc8fdebebe54a24"}, "graph":"https://api.thegraph.com/subgraphs/name/centrifuge/tinlake-kovan-staging", "contractConfig":{"JUNIOR_OPERATOR":"ALLOWANCE_OPERATOR","SENIOR_OPERATOR":"PROPORTIONAL_OPERATOR"}, "name":"Kovan Naxos Deployment","description":"This is a test description", "asset":"invoice" } ]
+NEXT_PUBLIC_POOLS_CONFIG_URL=https://raw.githubusercontent.com/centrifuge/tinlake-pool-config/master/kovan-staging.json?token=AA76YIGV5NI6OYOOCOLURQS645S3M
```

Comment:
What about importing the config instead through npm, as we do in tinlake.js?

Reply:
What we do right now needs a request on every page load - but the addresses could be statically included at build time.